airbyte-integrations/connectors/source-twilio/Dockerfile

```diff
@@ -1,16 +1,38 @@
-FROM python:3.9-slim
-
-# Bash is installed for more convenient debugging.
-RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
+FROM python:3.9.11-alpine3.15 as base
 
+# build and load all requirements
+FROM base as builder
 WORKDIR /airbyte/integration_code
+
+# upgrade pip to the latest version
+RUN apk --no-cache upgrade \
+    && pip install --upgrade pip \
+    && apk --no-cache add tzdata build-base
+
+
 COPY setup.py ./
-RUN pip install .
-COPY source_twilio ./source_twilio
+# install necessary packages to a temporary folder
+RUN pip install --prefix=/install .
+
+# build a clean environment
+FROM base
+WORKDIR /airbyte/integration_code
+
+# copy all loaded and built libraries to a pure basic image
+COPY --from=builder /install /usr/local
+# add default timezone settings
+COPY --from=builder /usr/share/zoneinfo/Etc/UTC /etc/localtime
+RUN echo "Etc/UTC" > /etc/timezone
+
+# bash is installed for more convenient debugging.
+RUN apk --no-cache add bash
+
+# copy payload code only
 COPY main.py ./
+COPY source_twilio ./source_twilio
 
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.5.0
+LABEL io.airbyte.version=1.0.0
```

Comment:
@erohmensing in some sense moving a python cdk to low-code won't change the behavior of the connector and it isn't a breaking change. I asked to be a major update to make clear about the drastic change into the code. Can you give your opinion here?

Reply:
@marcosmarxm can you take a look for any remaining changes to be made? I've migrated the behaviour completely.

Reply:
@marcosmarxm This is an interesting case. I understand the want to make it breaking if we fear potential breakage (I know we've had issues before where breaking changes are accidentally introduced in the migration due to e.g. catalog changes or spec changes

However if we do things right re: testing and being strict about refactors only being refactors (i.e. if the migration can't be performed without breaking something, maybe it's not a great candidate for low code yet), we wouldn't have to do this. 

I don't think I have a strong opinion either way - use your best judgement :)